webui/react/src/components/ExperimentChart.tsx

```diff
@@ -0,0 +1,140 @@
+import { Select } from 'antd';
+import { SelectValue } from 'antd/es/select';
+import Plotly, { PlotData } from 'plotly.js/lib/core';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+import { ValidationHistory } from 'types';
+import { clone } from 'utils/data';
+import { formatDatetime } from 'utils/date';
+import { capitalize, generateAlphaNumeric } from 'utils/string';
+
+import Section from './Section';
+import SelectFilter from './SelectFilter';
+
+const { Option } = Select;
+
+interface Props {
+  id?: string;
+  startTime?: string;
+  validationMetric?: string;
+  validationHistory?: ValidationHistory[];
+}
+
+enum Scale {
```

Comment:
suggestion: we'd probably use the scale selector for the trial charts but we can also refactor it then.

Reply:
👍  sounds good, let's refactor when trial charts is added.